spec.bs

```diff
@@ -107,8 +108,7 @@ dictionary AuctionAdInterestGroup {
 
 <div algorithm="joinAdInterestGroup()">
 
-The <dfn for=Navigator method>joinAdInterestGroup(|group|, |durationSeconds|)</dfn> method steps
-are:
+The <dfn for=Navigator method>joinAdInterestGroup(|group|)</dfn> method steps are:
 

```

Comment:
We should add a Note here about how we're going to slightly violate the specification until we finish the migration. Something like:

```suggestion
<div class="note">

Note: Temporarily, Chromium does not include the `required` keyword for {{AuctionAdInterestGroup/lifetimeMs}}, and instead starts this algorithm with the step

1. If |group|["lifetimeMs"] does not [=exist=], throw a {{TypeError}}.

This is detectable because it can change the set of fields that are read from the argument when a {{TypeError}} is eventually thrown, but it will never change whether the call succeeds or fails.

</div>
```

Reply:
Do we need spec notes for other places we've done a similar pattern? We've done the same technique when renaming renderUrl -> renderURL, etc.

Also, this note doesn't explain why we do this (to support the transition from the older API), not sure if it needs to? IIUC, we want to spec the behavior we will eventually have and not describe the older, deprecated versions that we'll eventually remove?

Reply:
Now that you're getting ready to send your I2S, I think we should try harder to document places that you know Chromium doesn't match the specification. It's not totally essential, and it's primarily worth doing in cases where someone might notice and think they need to file a bug or reverse engineer our behavior.

I thought about saying why, but I think it'll take more space than it's worth. Even this note is longer than ideal, if you see ways to shorten it. +1 for making the spec focus on where we're going rather than the deprecated versions.